packages/application-generic/src/usecases/create-execution-details/types/index.ts

```diff
@@ -8,6 +8,7 @@ export enum DetailEnum {
   STEP_CREATED = 'Step created',
   STEP_QUEUED = 'Step queued',
   STEP_DELAYED = 'Step delayed',
+  STEP_DIGESTED = 'Step digested',
```

Comment:
Added so it will be more informative because we could have a delay and digest delay. let me know if it makes sense.